lib/datadog/tracing/sampling/rule_sampler.rb

```diff
@@ -29,21 +29,29 @@ def initialize(
           default_sample_rate: Datadog.configuration.tracing.sampling.default_rate,
           default_sampler: nil
         )
```

Comment:
I think this section of the code will benefit from a comment explaining appsec & tracing interaction. Especially for someone who is new to both appsec and tracing.

Reply:
Thanks, I added a comment to explain it a bit more, let me know if you think that it is not comprehensive enough !